projects/1.18/assets/beyond-earth/beyond_earth/lang/zh_cn.json

```
@@ -0,0 +1,348 @@
+  "entity.beyond_earth.martian_raptor": "火星凶兽",
+  "entity.beyond_earth.rocket_t1": "一阶火箭",
+  "entity.beyond_earth.rocket_t2": "二阶火箭",
+  "entity.beyond_earth.rocket_t3": "三阶火箭",
+  "entity.beyond_earth.rocket_t4": "四阶火箭",
+  "entity.beyond_earth.lander": "着陆器",
+  "entity.beyond_earth.rover": "漫游车",
+
+  "biome.beyond_earth.orbit": "轨道",
+  "biome.beyond_earth.infernal_venus_barrens": "炼狱金星荒原",
+  "biome.beyond_earth.venus": "金星",
+  "biome.beyond_earth.mercury": "水星",
+  "biome.beyond_earth.mars": "火星",
+  "biome.beyond_earth.moon": "月球",
+  "biome.beyond_earth.mars_ice_spikes": "火星冰刺之地",
+  "biome.beyond_earth.glacio": "霜原",
+  "biome.beyond_earth.glacio_ice_spikes": "霜原冰刺之地",
+
+  "advancements.workbench_advancement.title": "NASA工作台",
+  "advancements.workbench_advancement.descr": "§b制造了一个万物的起源",
+  "advancements.rocket_advancement.title": "一阶火箭",
+  "advancements.rocket_advancement.descr": "§b 一个钢制火箭——真的能用?",
+  "advancements.rocket_tier_2_advancement.title": "二阶火箭",
+  "advancements.rocket_tier_2_advancement.descr": "§b 比钢制火箭多少好些...",
+  "advancements.rocket_tier_3_advancement.title": "三阶火箭",
+  "advancements.rocket_tier_3_advancement.descr": "§b 多么先进的科技!",
+  "advancements.rocket_tier_4_advancement.title": "四阶火箭",
+  "advancements.rocket_tier_4_advancement.descr": "§b 最为先进的火箭! 好耶!",
+  "advancements.moon_advancement.title": "不到月球非好汉",
+  "advancements.moon_advancement.descr": "§b一览月球真貌",
+
+  "effect.beyond_earth.oxygen_bubble_effect": "氧气泡",
+
+  "key.categories.beyond_earth": "超越地平",
+  "key.beyond_earth.rocket_start": "启动火箭",
+
+  "death.attack.oxygen": "%1$s 无法再呼吸了",
+  "death.attack.venus.acid": "%1$s 在酸雨中化为灰烬",
+
+  "entity.beyond_earth.alien.farmer": "农民",
+  "entity.beyond_earth.alien.fisherman": "渔民",
+  "entity.beyond_earth.alien.shepherd": "牧羊人",
+  "entity.beyond_earth.alien.fletcher": "制箭师",
+  "entity.beyond_earth.alien.librarian": "图书管理员",
+  "entity.beyond_earth.alien.cartographer": "制图师",
+  "entity.beyond_earth.alien.cleric": "牧师",
+  "entity.beyond_earth.alien.armorer": "盔甲匠",
+  "entity.beyond_earth.alien.weaponsmith": "武器匠",
+  "entity.beyond_earth.alien.toolsmith": "工具匠",
+  "entity.beyond_earth.alien.butcher": "屠夫",
+  "entity.beyond_earth.alien.leatherworker": "皮匠",
+  "entity.beyond_earth.alien.mason": "石匠",
+
+  "block.beyond_earth.molten_desh_fluid": "熔融戴斯",
+  "fluid.beyond_earth.molten_desh": "熔融戴斯",
+  "fluid.beyond_earth.flowing_molten_desh": "流动的熔融戴斯",
+  "item.beyond_earth.molten_desh_bucket": "熔融戴斯桶",
+  "block.beyond_earth.molten_ostrum_fluid": "熔融紫金",
+  "fluid.beyond_earth.molten_ostrum": "熔融紫金",
+  "fluid.beyond_earth.flowing_molten_ostrum": "流动的熔融紫金",
+  "item.beyond_earth.molten_ostrum_bucket": "熔融紫金桶",
+  "block.beyond_earth.molten_calorite_fluid": "熔融炽金",
+  "fluid.beyond_earth.molten_calorite": "熔融炽金",
+  "fluid.beyond_earth.flowing_molten_calorite": "流动的熔融炽金",
+  "item.beyond_earth.molten_calorite_bucket": "熔融炽金桶",
+
+  "container.beyond_earth.coal_generator": "火力发电机",
```

JEI 查看这个设备，只能给煤炭。
另外你术语能不能统一啊兄弟？
```suggestion
  "container.beyond_earth.coal_generator": "煤炭发电机",
```